### PR TITLE
Update set-up-atp-safe-links-policies.md

### PR DESCRIPTION
**Pull request by ruiwensu**
Clarifying reasons about why "Do not allow users to click through to original URL" option is suggested.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/set-up-atp-safe-links-policies.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ After you have reviewed (or edited) the default ATP Safe Links policy that appli
     
   - Select **Apply Safe Links to messages sent within the organization** if you would like to enable Safe Links for messages sent between users within your organization (recommended).
     
-  - Select **Do not allow user to click through to original URL**.
+  - Select **Do not allow user to click through to original URL** if you do not wish the individual users to override a *scan in progress* or *URL blocked* notification pages.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/set-up-atp-safe-links-policies.md]:
```suggestion
  - Select **Do not allow user to click through to original URL** if you do not wish the individual users to override a *scan in progress* or *URL blocked* notification page.
```

Comment (ruiwensu) [reply SecurityCompliance/set-up-atp-safe-links-policies.md]: But there are multiple versions of the notification pages. Scan in progress + 2 version of URL blocked (detected-blocked and admin-blocked). Should it be singular?

Comment (JohanFreelancer9) [reply SecurityCompliance/set-up-atp-safe-links-policies.md]:
```suggestion
  - Select **Do not allow user to click through to original URL** if you do not wish the individual users to override a *scan in progress* or *URL blocked* notification pages.
```
```suggestion
  - Select **Do not allow user to click through to original URL** if you do not wish the individual users to override *scan in progress* or *URL blocked* notification pages.
```

Comment (JohanFreelancer9) [reply SecurityCompliance/set-up-atp-safe-links-policies.md]: @ruiwensu Thanks for pointing that out. Please just remove the "a" as indicated.

**Comment by scanum** [comment]
@MSFTTracyP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: SecurityCompliance/set-up-atp-safe-links-policies.md (hunk 1 above)
Comment (MSFTTracyP) [reply SecurityCompliance/set-up-atp-safe-links-policies.md]: Either page (people are sitting on a page not on pages for the click) or pages (2 different error pages) works in this context.

**Pull request merged**